packages/grid/x-data-grid/src/tests/keyboard.DataGrid.test.tsx

```diff
@@ -536,9 +536,9 @@ describe('<DataGrid /> - Keyboard', () => {
     const input = screen.getByTestId('custom-input');
     input.focus();
     fireEvent.keyDown(input, { key: 'a' });
-    expect(renderCell.callCount).to.equal(4);
+    expect(renderCell.callCount).to.equal(6);
```

Comment:
Why was this test impacted by the changes? It only renders one row, so it should not be impacted by virtualization